types/index.d.ts

```diff
@@ -668,7 +669,7 @@ declare module 'cloudinary' {
                 image_metadata: object;
                 faces: number[][];
                 quality_analysis: number;
-                colors: string[][];
+                colors: [`#${string}`, number][];
```

Comment:
@karlhorky Looks like there's another usage here.

Reply:
Ah sorry, forgot that I updated two lines in this PR. Fixed in f4e57998c0196ca0aa9b13cff8c74a349a73aeeb